src/moduler/aktivitet/visning/referat/oppdater-referat-form.js

```diff
@@ -75,9 +81,27 @@ function OppdaterReferatForm(props) {
                 Del med bruker
             </HiddenIfHovedknapp>
 
-            <Knapp spinner={oppdaterer} disabled={oppdaterer}>
-                Lagre utkast
-            </Knapp>
+            <>
+                {erReferatPublisert ? (
```

Comment:
Er en smule rart å rendre noe slikt. Er da litt bedre å lage en ny komponent som enten returnerer den ene eller den andre. Hvorfor byttes knapp fra `del endring` til `lagre utkast` ?

Reply:
Siden hvis man har allerede delt et samtalereferat med en bruker så er det ikke lenger et utkast man skriver på, og den endringen man da gjør vil bli _delt_ med brukeren. Er referatet ikke publisert, så er det fremdeles et utkast, derfor 'lagre utkast'. Men kan godt lage en ny komponent, men da blir det bare for å gjøre koden enklere å lese her, for det vil jo bli akkurat samme logikk innad i den nye komponenten?